packages/ruby/spec/readme/filter_spec.rb

```diff
@@ -0,0 +1,59 @@
+require "readme/filter"
+
+RSpec.describe Filter do
+  describe ".for" do
+    it "returns RejectParams when only reject argument is given" do
+      result = Filter.for(reject: ["reject"])
+      expect(result.class).to eq Filter::RejectParams
+    end
+
+    it "returns AllowOnly when only allow_only argument is given" do
+      result = Filter.for(allow_only: ["keep"])
+      expect(result.class).to eq Filter::AllowOnly
+    end
+
+    it "returns None when neither arugment is given" do
+      result = Filter.for
+      expect(result.class).to eq Filter::None
+    end
```

Comment:
A slightly RSpec way to write these expectations would be:

```ruby
expect(result).to be_an_instance_of Filter::None
```

Reply:
Nice, I'll update this